docs/initial-sync.md

```diff
@@ -0,0 +1,58 @@
+# File copy based initial sync
+
+!!! admonition "Version added: [7.0.21-12](release_notes/7.0.21-12.md)"
+
+When a new member joins the replica set, it receives the data from the existing replica set node via the initial sync. 
+
+In [Percona Server for MongoDB Pro](psmdb-pro.md), you can choose a file copy-based initial sync for a new node. You must have WiredTiger defined as the storage.
+
+File copy-based initial sync method is a physical copying of the data files from the source to the target. It is much faster than the default [logical initial sync :octicons-link-external-16:](https://www.mongodb.com/docs/manual/core/replica-set-sync/#logical-initial-sync-process), which is especially beneficial in heavy write environments. Using this sync method speeds up cluster scaling and increases restore performance. 
+
+File copy-based initial sync implementation is compatible with that of MongoDB Advanced and has the same [configuration parameters](#configuration-parameters).
+
+To select the initial sync method, specify the following configuration in the configuration file for the target node:
+
+```yaml
+setParameter:
+  initialSyncMethod: fileCopyBased
+```
+
+You can only set this configuration at startup.
+
+## Workflow
+
+When you start a new node for the replica set, the workflow is the following:
+
+1. The new node (also referred to as the target node) selects for the source node for the sync. This sync source is typically the node that responded first and has the passing configuration (e.g. it has WiredTiger set as the storage and the same arrangement of files and indexes as the target node.)
+2. The target node opens a `$backupCursor` towards the sync source. The `$backupCursor` returns the list of files to copy and the timestamp of the oplog end in the metadata file.
+3. The file copy starts. During this process the target node lags behind the sync source as it remains operational and its data changes. The sync source node is periodically checked to ensure the time of the lag falls within the dined time.
+4. If the lag between the sync source differs and the target exceeds the defined threshold, the target node opens the `$backupCursorExtended` to retrieve the changes. Depending on the file copy duration, the target node can open `$backupCursorExtended` several times, limited by the maximum number of cycles (3 by default)
```

Comment:
> If the lag between the sync source differs and the target exceeds the defined threshold

The "diiffers" should be removed

Reply:
I think it would be better to say that target node "executes" $backupCursorExtend instead of "opens"